pkg/dogstatsd/seelog_config.go

```diff
@@ -0,0 +1,99 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package dogstatsd
+
+import (
+	"bytes"
+	"html/template"
+	"strings"
+	"sync"
+)
+
+// Config abstracts seelog XML configuration definition
+type Config struct {
+	settings map[string]interface{}
+	sync.Mutex
+}
+
+const seelogConfigurationTemplate = `
+<seelog minlevel="{{.logLevel}}">
```

Comment:
This file is mostly Copy/Pasted from https://github.com/DataDog/datadog-agent/blob/main/pkg/config/internal/seelog/seelog_config.go

I think we can improve this by either: 
1. making `seelog/seelog_config.go` non-internal (since we now want to use it in more than one place)
2. Or - trim this file down to just the necessary bits. In fact, you probably don't need the majority of what is defined here and can just inline it with your implementation (where you call `NewSeelogConfig` and `Render`)

Reply:
I opted into moving the `seelog_config.go` outside of the internal folder and changed other files import that were using the internal file